apps/coscrad-frontend/src/emotion-components/theme.ts

```diff
@@ -0,0 +1,18 @@
+import styled from '@emotion/styled';
```

Comment:
Not sure why `src/emotion-components/table.ts` is empty. Was there something you wanted to experiment with there?